components/brave_wallet/browser/eth_data_builder.cc

```diff
@@ -59,6 +59,42 @@ absl::optional<std::string> ChainIdToVersion(const std::string& symbol,
 
 }  // namespace
 
+namespace filforwarder {
+
+absl::optional<std::string> Forward(const std::vector<uint8_t>& destination) {
+  auto function_hash = GetFunctionHash("forward(bytes)");
+  size_t destination_padding = 32;
+  size_t destination_size = destination.size();
+  std::string padded_padding;
+  std::string padded_dest_size;
+
+  if (!PadHexEncodedParameter(Uint256ValueToHex(destination_padding),
+                              &padded_padding)) {
+    return absl::nullopt;
+  }
+
+  if (!PadHexEncodedParameter(Uint256ValueToHex(destination_size),
+                              &padded_dest_size)) {
+    return absl::nullopt;
+  }
+
+  std::string padded_destination;
+  auto dest = brave_wallet::ToHex(destination);
+  if (!PadHexEncodedParameter(dest, &padded_destination, false)) {
+    return absl::nullopt;
+  }
+  std::vector<std::string> hex_strings = {function_hash, padded_padding,
+                                          padded_dest_size, padded_destination};
+  std::string data;
+  if (!ConcatHexStrings(hex_strings, &data)) {
+    return absl::nullopt;
+  }
+
+  return data;
+}
+
+}  // namespace filforwarder
+
```

Comment:
I'd suggest implementing this with different utility like here
https://github.com/brave/brave-core/blob/master/components/brave_wallet/browser/ens_resolver_task.cc#L419-L422
```
  auto forwarder_call = eth_abi::TupleEncoder()
                              .AddBytes(fil_address_bytes)
                              .EncodeWithSelector(kFilForwarderTransferSelector);
  return ToHex(forwarder_call);
```